qa/rpc-tests/wallet_sapling.py

```diff
@@ -136,11 +136,13 @@ def run_test(self):
 
         # Verify importing a spending key will update the nullifiers and witnesses correctly
         sk0 = self.nodes[0].z_exportkey(saplingAddr0)
-        self.nodes[2].z_importkey(sk0, "yes")
-        assert_equal(self.nodes[2].z_getbalance(saplingAddr0), Decimal('10'))
+        saplingAddr0 = self.nodes[2].z_importkey(sk0, "yes")
```

Comment:
This shadows the original `saplingAddr0`, functionally altering it from an address to a tuple containing an address, which is likely to confuse future people modifying this test. Use a different name for the returned tuple.